apps/web/src/styles/index.css

```diff
@@ -1,3 +1,5 @@
+@import url('https://fonts.googleapis.com/css2?family=Overpass&display=swap');
+
```

Comment:
imported Overpass font used in MantineOverride

Reply:
Done checking. and it is working on my end. Thanks for this!